Sources/Paywalls/PaywallData.swift

```diff
@@ -62,13 +64,21 @@ extension PaywallData {
         public var title: String
         public var subtitle: String
         public var callToAction: String
-        public var callToActionWithIntroOffer: String?
         public var offerDetails: String
-        public var offerDetailsWithIntroOffer: String?
-
-        // swiftlint:enable missing_docs
+        @NonEmptyStringDecodable
+        var _callToActionWithIntroOffer: String?
```

Comment:
These can't be `public` because the property wrapper isn't, so they get exposed separately from the property that's decoded/encoded.